### PR TITLE
Fix Issue#7818 for not being able to update address for guest checkou…

### PR DESCRIPTION
**Pull request by charleswong28**
Fixing the issue #7818 by creating a hidden radio box in customer detail view to pass guest_checkout parameter.

### CODE REVIEW

File: backend/app/views/spree/admin/orders/customer_details/_form.html.erb [hunk 1]
```diff
@@ -27,6 +27,7 @@
               <div>
                 <span class="badge">
                   <%= @order.user.nil? ? Spree.t(:say_yes) : Spree.t(:say_no) %>
+                  <%= radio_button_tag :guest_checkout, true, @order.user.nil?, class: 'hide' %>
```
Comment (vishalzambre) [inline backend/app/views/spree/admin/orders/customer_details/_form.html.erb]: @charleswong28  I think Input hidden is the right input tag for this

```
 <%= hidden_field_tag :guest_checkout, 'true' if @order.user.nil? %>
```

**Review by charleswong28 — COMMENTED** [review]
You're right. My bad.

**Comment by jafffmen** [comment]
@charleswong28 could you resolve conflicts and push rebased branch please? 🙂

**Comment by nnande** [comment]
@charleswong28 the changes are already introduced to 3-1-stable via https://github.com/spree/spree/commit/c8aba60add323f8451c00cc47dfed702c58458bb

Also please always make PRs against master branch.
https://github.com/spree/spree/blob/master/.github/CONTRIBUTING.md  (p. 8)

**Pull request closed without merging**